vunit/sim_if/rivierapro.py

```diff
@@ -210,7 +210,10 @@ def compile_verilog_file_command(self, source_file):
         for include_dir in source_file.include_dirs:
             args += ["+incdir+%s" % include_dir]
         for key, value in source_file.defines.items():
-            args += ["+define+%s=%s" % (key, value)]
+            if value != "":
```

Comment:
What do you think about the following?

```py
args += ["+define+%s" % key]
if value:
    args[-1] += "=%s" % value
```

Also, please, run `tox -e py38-fmt` for getting rid of CI errors.

Reply:
I've updated code with your solution. Seems like it passes in tox tests now

Reply:
Yes. Errors were produced because of an indentation issue. Your code was otherwise correct too. The change was just a style preference, since it had to be edited.

Thanks!